src/app/client/client.component.ts

```diff
@@ -31,9 +32,20 @@ export class ClientComponent implements OnInit {
   ngOnInit() {
     this.client = new Client();
     this.client.categorie = new Categorie();
-    if (this.sharedService.currentClient) {
-      this.client = this.sharedService.currentClient;
-      this.title = 'Modifiez vos informations personelles';
+    // If a client is currently logged
+    if (this.sharedService.getCurrentClient()) {
+      // We have to reload client each time we get to the page
+      // It is to avoid having previous information types into from
```

Comment:
Into form you mean ? :p

Reply:
Le commentaire ne veut affectivement rien dire #fatigue
J'imagine que ça devait être un truc genre "It is to avoid having previous information typed and not validated to be into the form"